helm/content-store/values.yaml

```diff
@@ -10,17 +10,21 @@ service:
   annotations: {}
   type: NodePort
   port: 80
-  targetPort: 80
-
 
 replicaCount: 1
 
-image:
+appImage:
   repository: 172025368201.dkr.ecr.eu-west-1.amazonaws.com/content-store
   pullPolicy: IfNotPresent
   tag: latest
 
-containerPort: 80
+nginxImage:
+  repository: nginx
+  pullPolicy: IfNotPresent
```

Comment:
Something to be aware of here - if `imageTag` is `latest` and `pullPolicy` is `IfNotPresent` then Kubelet on the node will pull the image the first time then never pull it again, unless either the tag or pull policy is changed at a later date. This will also apply to other unrelated Pods or Deployments that land on the same worker that also use the same `image:tag` and `pullPolicy: IfNotPresent` combo, as from Kubernetes' point an image tagged with `latest` is indeed already `Present`.

This is one reason why it's recommended to [not use `latest`](https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy), because it's non-deterministic; or, to specify `imagePullPolicy: Always`, so that the latest image is always pulled on deploy. There are pros and cons to both pinning a specific tag and always pulling `latest`, but generally I'd lean towards being specific, so that we don't get an unexpected breaking change from a new major version one day.

tl;dr - I'd change `tag` to `1.21.1`

Reply:
Snap :) If you go with a specific version then we'll need an update mechanism though (which we'll want eventually but it'll probably be a while). We could go with `stable` in the interim so at least we'll get security updates.

Reply:
Sorry I'm being an idiot - that'll have the same issue. The interim thing until we have an update mechanism would be to keep pullPolicy: Always in conjunction with using `stable`.

Reply:
Yeah, either approach is fine in this case I think, particularly as a specific tag can always be provided at `helm install` time - a default of `stable` with `Always` seems reasonable 👍

Reply:
So it's either `pullPolicy: Always` and `stable` or `1.20.1` and no/default PullPolicy and a big scary TODO to figure out a Dependabot-like thing to update the pinned versions.

Reply:
> particularly as a specific tag can always be provided at helm install time
Ah yeah good point

Reply:
Yeah go with `stable / Always` - it's a reasonable default when nothing specific has been requested at `helm install` time.